src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1224,28 +1239,26 @@ static String getRanges(JsonArray ar, String startProp, String endProp) {
    *   WHERE daterange($3, $4) @> lower(usageDateRange) AND openAccess
    *   GROUP BY titleEntryId
    * ) t1 ON t1.titleEntryId = title_entries.id
-   * WHERE agreementId = $1;
+   * WHERE agreementId = $1
+   *   AND NOT (printISSN IS NULL AND onlineISSN IS NULL)
```

Comment:
Surely this comment is wrong in the BOOK case?

Reply:
This comment lists _one_ example, this is sufficient.
The SQL changes when changing the number of periods (n0, n1, n2, ...), openAccess (true/false), or report (journal, book).

Reply:
Gotcha.